independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/extension-base/java/runtime/pom.tpl.qute.xml

```diff
@@ -13,6 +13,7 @@
     <artifactId>{namespace.id}{extension.id}</artifactId>
     {#if extension.name}
     <name>{namespace.name}{extension.name} - Runtime</name>
```

Comment:
My worry is that people will keep that description when releasing.. maybe we could have a check to make sure it's different from the name somewhere?